desktop/src/cli.rs

```diff
@@ -195,6 +195,12 @@ pub struct Opt {
         value_name = "GAMEPAD BUTTON>=<KEY NAME"
     )]
     pub gamepad_button: Vec<(GamepadButton, KeyCode)>,
+
+    /// Disable AVM2 optimizer.
+    /// Note that some early opcode conversions
+    /// (like inlining constant pool entries) can't be disabled.
+    #[clap(long)]
+    pub no_avm2_optimizer: bool,
```

Comment:
I know it's late, and not a big deal... But I have a similar situation going on with the disablization of OpenH264 over at #14654. And I deliberately chose the "positive meaning" to be the flag, and set the default value to `true`.
Has this been considered and decided against here, or was it done just so `false` can be kept as the default flag value?